### PR TITLE
docs(flutter): Document Android 16KB Page Size

### PR DESCRIPTION
**Pull request by denrase**
Document Android 16KB Page Size

- Alert in index
- Section in troubleshooting guide

## EXTRA RESOURCES

- [sentry-dart 16kb issue](https://github.com/getsentry/sentry-dart/issues/2237)

### CODE REVIEW

**Comment by coolguyzone** [comment]
@buenaflor please take a look at this one 👁️

File: docs/platforms/flutter/index.mdx [hunk 1]
```diff
@@ -88,6 +88,11 @@ try {
 }
 ```
 
+<Alert level="warning" title="Android 16 KB Page Size">
+  Use Sentry Flutter SDK version `8.11.0` and above to support 16 KB page sizes on Andorid devices, or your app will crash.
```
Comment (buenaflor) [inline docs/platforms/flutter/index.mdx:92]:
```suggestion
  Use Sentry Flutter SDK version `8.11.0` and above to support 16 KB page sizes on Android devices, or your app will crash.
```

File: docs/platforms/flutter/troubleshooting.mdx [hunk 2]
```diff
@@ -33,6 +33,14 @@ flutter build apk --target-platform=android-arm --split-per-abi
 
 The configuration for symbolication of Native events (C/C++) is documented in our [Android Native Development Kit](/platforms/android/configuration/using-ndk/) content. If you are having issues with symbolication in Flutter, check that your configuration is correct, as discussed in our Flutter content that covers [Uploading for Android NDK](/platforms/flutter/upload-debug/#uploading-for-android-ndk)
 
+## Support 16 KB Page Sizes on Android
+
+Starting with Android 15, AOSP supports devices with a 16 KB page size. If your app uses NDK libraries (directly or via an SDK), you’ll need to rebuild it for compatibility with these devices.
+
+Update to Sentry Flutter SDK version `8.11.0` and above order to support 16 KB page sizes on Andorid devices. 
+
+Please read the [Andorid developer documentation](https://developer.android.com/guide/practices/page-sizes) to update and test your app with 16 KB page size support.
```
Comment (buenaflor) [inline docs/platforms/flutter/troubleshooting.mdx:42]:
```suggestion
Update to Sentry Flutter SDK version `8.11.0` and above order to support 16 KB page sizes on Android devices. 

Please read the [Android developer documentation](https://developer.android.com/guide/practices/page-sizes) to update and test your app with 16 KB page size support.
```

**Review by buenaflor — CHANGES_REQUESTED** [review]
Looks good, 2 small changes, I won't approve yet because `8.11.0` is not out

**Comment by buenaflor** [comment]
@coolguyzone can we trigger the deployment pls

File: docs/platforms/flutter/index.mdx [hunk 3]
```diff
@@ -88,6 +88,11 @@ try {
...
+  Refer to [Troubleshooting](/platforms/flutter/troubleshooting/#support-16-kb-page-sizes-on-android) for more information.
+</Alert>
+
```
Comment (buenaflor) [inline docs/platforms/flutter/index.mdx:95]: I wonder if we need this in the getting started, in the preview it looks like we're 'polluting' the getting started page.

imo it's fine to only keep it in troubleshoot

Comment (kahest) [reply docs/platforms/flutter/index.mdx:95]: both is fine for me, but we should move the troubleshooting entry to the top IMO

Comment (buenaflor) [reply docs/platforms/flutter/index.mdx:95]: 👍 

@denrase let's remove it from the getting started page and move it to the top of the troublehsooting page

**Review by kahest — APPROVED** [review]

**Review by buenaflor — APPROVED** [review]

**Pull request merged**